ui/src/common/components/__tests__/__snapshots__/SafeSwitch.test.jsx.snap

```diff
@@ -1,13 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`SafeSwitch renders childrens and redirect to errors 1`] = `
-<Switch>
-  <Route
-    component={[Function]}
-    path="/foo"
-  />
-  <Redirect
-    to="/errors"
-  />
-</Switch>
-`;
+exports[`SafeSwitch renders childrens and redirect to errors 1`] = `<DocumentFragment />`;
```

Comment:
The test corresponding to this snapshot is not much of a use. Either we find a better way to test it (if it makes sense) or remove the test